src/mame/drivers/rltennis.cpp

```diff
@@ -74,11 +74,11 @@ player - when there's nothing to play - first, empty 2k of ROMs are selected.
 
 #define RLT_REFRESH_RATE   60
 #define RLT_TIMER_FREQ     (RLT_REFRESH_RATE*256)
-#define RLT_XTAL           XTAL(12'000'000)
+#define RLT_XTAL           XTAL(32'000'000)
 
 uint16_t rltennis_state::io_r()
 {
-	return (ioport("P1" )->read()&0x1fff) | (m_unk_counter<<13); /* top 3 bits controls smaple address update */
+	return (ioport("P1" )->read()&0x1fff) | (m_unk_counter<<13); // Top 3 bits controls smaple address update
```

Comment:
If you're reformatting the comment, you should probably fix the spelling error and grammar error while you’re causing diff noise on the line.

Reply:
Fixed, thanks!

Reply:
Not quite – remember English words for plurals: "He controls the mafia," vs "They control the mafia."

Reply:
Thanks again!